x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/apikey/ApiKeyRestIT.java

```diff
@@ -54,7 +55,7 @@ public void createUsers() throws IOException {
         createUser(SYSTEM_USER, SYSTEM_USER_PASSWORD, List.of("system_role"));
         createRole("system_role", Set.of("grant_api_key"));
         createUser(END_USER, END_USER_PASSWORD, List.of("user_role"));
-        createRole("user_role", Set.of("monitor"));
+        createRole("user_role", Set.of("monitor", "manage_own_api_key"));
```

Comment:
I'd prefer we have a different role and user for the new test since it is better for the existing grantApiKey tests to keep having a user without any API Key related privileges. It's not absolutely necessary since we know today it should not make a difference but it feels more future proof.